packages/block-library/src/audio/edit.js

```diff
@@ -128,6 +129,22 @@ function AudioEdit( {
 	} );
 
 	if ( ! src && ! temporaryURL ) {
+		const placeholder = ( content ) => {
+			return (
+				<Placeholder
+					className="block-editor-media-placeholder"
+					withIllustration
```

Comment:
```suggestion
					withIllustration={ ! isSingleSelected }
```

How about enabling the `withIllustration` prop only when the block is not selected? That way, it would seem like we wouldn't need any CSS to override it.